### ISSUE TITLE
pytest-testdox repo is not testable after installing `pytest-testdox`

### ISSUE THREAD

**Issue by RomuloOliveira** [renanivo/pytest-testdox#13]
After installing `pytest-testdox` via pip, I'm not able anymore to run `make test`.

Working environment:

```
 ⟩ pip freeze
appdirs==1.4.3
bumpversion==0.5.3
coverage==4.3.4
flake8==3.3.0
isort==4.2.5
mccabe==0.6.1
mock==2.0.0
packaging==16.8
pbr==3.0.0
py==1.4.33
pycodestyle==2.3.1
pyflakes==1.5.0
pyparsing==2.2.0
pytest==3.0.7
pytest-cov==2.4.0
six==1.10.0

~/projects/pytest-testdox (pytest-testdox) (master)
 ⟩ make test
============================================================ test session starts ============================================================
platform darwin -- Python 3.5.2, pytest-3.0.7, py-1.4.33, pluggy-0.4.0
rootdir: /Users/romulo/projects/pytest-testdox, inifile:
plugins: cov-2.4.0
collected 26 items

tests/test_formatters.py .............
tests/test_models.py ......
tests/test_plugin.py .......

---------- coverage: platform darwin, python 3.5.2-final-0 -----------
Coverage XML written to file coverage.xml


========================================================= 26 passed in 0.91 seconds =========================================================
```

And then:

```
 ⟩ pip install pytest-testdox --upgrade
Collecting pytest-testdox
  Using cached pytest_testdox-0.2.1-py2.py3-none-any.whl
Requirement already up-to-date: pytest>=3.0.0 in /Users/romulo/.virtualenvs/pytest-testdox/lib/python3.5/site-packages (from pytest-testdox)
Requirement already up-to-date: py>=1.4.29 in /Users/romulo/.virtualenvs/pytest-testdox/lib/python3.5/site-packages (from pytest>=3.0.0->pytest-testdox)
Requirement already up-to-date: setuptools in /Users/romulo/.virtualenvs/pytest-testdox/lib/python3.5/site-packages (from pytest>=3.0.0->pytest-testdox)
Requirement already up-to-date: packaging>=16.8 in /Users/romulo/.virtualenvs/pytest-testdox/lib/python3.5/site-packages (from setuptools->pytest>=3.0.0->pytest-testdox)
Requirement already up-to-date: six>=1.6.0 in /Users/romulo/.virtualenvs/pytest-testdox/lib/python3.5/site-packages (from setuptools->pytest>=3.0.0->pytest-testdox)
Requirement already up-to-date: appdirs>=1.4.0 in /Users/romulo/.virtualenvs/pytest-testdox/lib/python3.5/site-packages (from setuptools->pytest>=3.0.0->pytest-testdox)
Requirement already up-to-date: pyparsing in /Users/romulo/.virtualenvs/pytest-testdox/lib/python3.5/site-packages (from packaging>=16.8->setuptools->pytest>=3.0.0->pytest-testdox)
Installing collected packages: pytest-testdox
Successfully installed pytest-testdox-0.2.1
~/projects/pytest-testdox (pytest-testdox) (master)
 ⟩ make test
============================================================ test session starts ============================================================
platform darwin -- Python 3.5.2, pytest-3.0.7, py-1.4.33, pluggy-0.4.0
rootdir: /Users/romulo/projects/pytest-testdox, inifile:
plugins: testdox-0.2.1, cov-2.4.0
collected 20 items / 1 errors
Coverage.py warning: No data was collected.


---------- coverage: platform darwin, python 3.5.2-final-0 -----------
Coverage XML written to file coverage.xml

================================================================== ERRORS ===================================================================
___________________________________________________ ERROR collecting tests/test_models.py ___________________________________________________
ImportError while importing test module '/Users/romulo/projects/pytest-testdox/tests/test_models.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_models.py:7: in <module>
    from pytest_testdox.models import Node, PatternConfig, Result
E   ImportError: No module named 'pytest_testdox.models'
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
========================================================== 1 error in 0.45 seconds ==========================================================
make: *** [test] Error 2
```

I guess its because of _entry_points_, but I did not tested any solution yet.

```
entry_points={
        'pytest11': [
            'testdox = pytest_testdox.plugin',
        ],
    },
```

**Comment by renanivo** [comment]
I think if we used relative imports in tests, it would work for most of it. But, the plugin tests would still fail, because of what you wrote: Pytest would look for the testdox plugin in `pytest_testdox.plugin` and would find the module in `site-packages` before the current version of code (at the current working directory). That's why I had to change the way the tests loaded the plugin in the PR #7.

Meanwhile we'll have to uninstall `pytest-testdox` from our virtualenvs in order to the tests run.
